nmostesting/suites/IS0702Test.py

```diff
@@ -513,6 +669,79 @@ def get_websocket_connection_sources(self, test):
                                             connection_sources[connection_uri].append(self.is04_sources[source_id])
         return connection_sources
 
+    def get_mqtt_broker_senders(self, test):
+        """Returns a dictionary of MQTT senders available for connection"""
+        broker_senders = {}
+
+        if len(self.is07_sources) > 0:
```

Comment:
Just trying to avoid a little of the indentation here, could this top level if statement be removed given there's no 'else' and the next step is a for loop over the same list?

Reply:
Ha, yes, I think that can be applied to this and the `get_websocket_connection_sources` above on which I assume it was based...